sphinx/user-guide/installation.rst

```diff
@@ -0,0 +1,427 @@
+.. include:: ../hyperlinks.rst
+   :start-line: 1
+
+Installation
+============
+
+.. _archives: https://github.com/metomi/rose/tags
+
+The source code for Rose is available via `github <rose github>`_, code releases
+are available in ``zip`` and ``tar.gz`` `archives`_.
+
+1. Un-pack the archive file into an appropriate location on your system.
+2. Add the ``rose/bin/`` directory into your ``PATH`` environment variable.
+3. Check system compatability by running ``rose check-software``.
+
+
+System Requirements
+-------------------
+
+Rose runs on Unix/Linux systems and is known to run on RHEL6 and a number of
+systems including those documented under `metomi-vms`_.
+
+System compatibility can be tested using the ``rose check-software`` command.
+
+.. script-include:: rose check-software --rst
+
+
+Host Requirements
+-----------------
+
+Whilst you can install and use Rose & cylc on a single host (i.e. machine),
+most installations are likely to be more complex. There are five types of
+installation:
+
+Hosts For Running Cylc Suites
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Each user can just run their suites on the host where they issue the
+``rose suite-run`` command. However, the local host may not meet the
+necessary requirements for connectivity (to the hosts running the tasks)
+or for availability (the host needs to remain up for the duration of
+the suite). Therefore, Rose can be configured to run the suites on
+remote hosts. If multiple hosts are available, by default the least loaded
+of the available hosts will be used.
+
+Installation requirements:
+   * Rose, cylc, Bash, Python, jinja2.
+   * Subversion & FCM *(only if you want rose suite-run to install files
+     from Subversion using FCM keywords).*
+Connectivity requirements:
+   * Must be able to submit tasks to the hosts which run the suite tasks,
+     either directly or via SSH access to another host.
+
+Hosts For Running Tasks In Suites
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Installation requirements:
+   * Rose, cylc, Bash, Python.
+   * Subversion & FCM (only if you want to use the Rose install utility
```

Comment:
Perhaps pop the bracketed text in italics to be consistent with the equivalent statement for `Hosts For Running Cylc Suites` (line 48)?